ci/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG TENDERMINT_VERSION=0.34.9
+ARG GAIA_VERSION=4.2.1
+ARG RUST_VERSION=1.57
+
+FROM tendermint/tendermint:v${TENDERMINT_VERSION} AS tendermint
+FROM cephalopodequipment/gaiad:${GAIA_VERSION} AS gaia
+FROM rust:${RUST_VERSION}-slim-bullseye
+
+COPY --from=tendermint /usr/bin/tendermint /usr/bin/tendermint
+COPY --from=gaia /usr/bin/gaiad /usr/bin/gaiad
+
+ENV IBC_SRC=/src/ibc-rs
+ENV BASECOIN_SRC=/src/basecoin-rs
+ENV LOG_DIR=/var/log/basecoin-rs
+ENV IBC_COMMITISH=master
```

Comment:
FYI: If it proves problematic to test with `ibc-rs:master`, we can eventually move to using the docker images, e.g.,

https://hub.docker.com/r/informalsystems/hermes/tags

This will also speed-up the CI job, as Hermes is built-in the image.

Reply:
Good idea! I've logged #36 to implement this in a follow-up issue :+1: